2.1a_Python_for_HPC/dask_slurm/dask_workers.slrm

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 #SBATCH --job-name="dask-workers"
 #SBATCH --output="dask-workers.%j.%N.out"
-#SBATCH --partition=compute
-#SBATCH --account=sds166
-#SBATCH --nodes=2
-#SBATCH --ntasks-per-node=24
+### We are not using GPUs but we have them reserved, so let's
+### run dask workers on the GPU nodes
+#SBATCH --partition=gpu-shared
+#SBATCH --gpus=2
+#SBATCH --account=crl155
+##SBATCH --account=sds166
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=2
+#SBATCH --cpus-per-task=20
 #SBATCH --export=ALL
 #SBATCH -t 0:20:00
 ### REMOVE RESERVATION AFTER THE SUMMER INSTITUTE
-###SBATCH --reservation=si2020resday2
+#SBATCH --reservation=SI2021RES
 
-srun --ntasks-per-node=1 bash ./launch_worker.sh 
+srun bash ./launch_worker.sh
```

Comment:
@mkandes @mahidhar I'm launching the 2 dask workers on the GPU nodes in the shared partition, half node for each user. can you also please double-check here?